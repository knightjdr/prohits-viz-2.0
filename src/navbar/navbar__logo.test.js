import React from 'react';
import { shallow } from 'enzyme';

import Logo from './navbar__logo';

describe('Navbar Logo', () => {
  it('should match snapshot when the navbar has a background', () => {
    const wrapper = shallow(<Logo background />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should match snapshot when the navbar has no background', () => {
    const wrapper = shallow(<Logo background={false} />);
    expect(wrapper).toMatchSnapshot();
  });
});
