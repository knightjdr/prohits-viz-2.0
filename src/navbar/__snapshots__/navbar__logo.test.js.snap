// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Logo should match snapshot when the navbar has a background 1`] = `
<RouterLink
  className="navbar__logo-link"
  to="/"
>
  <img
    alt="ProHits-viz logo"
    className="navbar__logo"
    src="logo_accent_nobg.svg"
  />
</RouterLink>
`;

exports[`Navbar Logo should match snapshot when the navbar has no background 1`] = `
<RouterLink
  className="navbar__logo-link"
  to="/"
>
  <img
    alt="ProHits-viz logo"
    className="navbar__logo"
    src="logo_nobg.svg"
  />
</RouterLink>
`;
